Week3/db-week-3.js

```diff
@@ -0,0 +1,216 @@
+'use strict';
+const util = require('util');
+const fs = require('fs');
+const mysql = require('mysql');
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const port = 3020;
+
+const todoDatabase = {
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'todo',
+  multipleStatements: true,
+};
+
+app.set('view engine', 'pug');
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(express.static(__dirname + '/public'));
+
+const CREATE_TABLE_TODOS = `CREATE TABLE IF NOT EXISTS todos ( 
+  todoid VARCHAR(30) PRIMARY KEY,
+  reminder text 
+  );`;
+
+const CREATE_TABLE_ITEMS = `CREATE TABLE IF NOT EXISTS items (
+    item VARCHAR(30),
+    isdone enum('Yes', 'No'),
+    belongs_to VARCHAR(30),
+    
+    FOREIGN KEY (belongs_to)
+    REFERENCES todos(todoid)
+    ON DELETE CASCADE
+    );`;
+
+app.get('/', async function(req, res) {
+  try {
+    const connection = mysql.createConnection(todoDatabase);

```

Comment:
No need to change it, but there's also this option https://stackoverflow.com/questions/37102364/how-do-i-create-a-mysql-connection-pool-while-working-with-nodejs-and-express to use it across your code. Might be worth looking into it for future projects :)